bintray.gradle

```diff
@@ -0,0 +1,26 @@
+//
+// This file is to be applied to some subproject.
+//
+
+apply plugin: 'com.jfrog.bintray'
+
+bintray {
+    user = bintray_user
+    key = bintray_api_key
+    configurations = ['archives']
+    pkg {
+        repo = 'org.jmonkeyengine'
```

Comment:
Why not use the POM_ constants for everything? We want the POMs in our repo and the internal repo to be as similar as possible